docs/TEAM.md

```diff
@@ -6,7 +6,7 @@
 |[Adewale Abiola]( https://github.com/princehaybee )|I am a Nigerian. I have basic knowledge of HTML,CSS and Javascript.|
 |[Micheal Aderinto](https://github.com/ad3rinto)| Manchester UK, Hoping this will be my first step in the world of solving world problems with code.|
 |Usha Kiran ([K!r@nD0n](https://twitter.com/ushakiran_m)) | I am a student from India. I just started my OpenSource journey with TWINDLE. I am currently learning #JavaScript.|
-|Eyram ([Ey_kwasi](https://twitter.com/Ey_kwasi))| I am from Ghana and I have no experience, started learning web development some months ago. I am participating to learn and also make twindle a success.|
+|<img src="docs\images\team\eyram.jpg" >Eyram ([Ey_kwasi](https://twitter.com/Ey_kwasi))| I am from Ghana and I have no experience, started learning web development some months ago. I am participating to learn and also make twindle a success.|
```

Comment:
@EYRAM565 Please match your format with [this](https://github.com/twindle-co/twindle/pull/451/files#diff-35d64f6afeed8fc6d1c2263211980cbb6c2ec53ae38a9e471881b31e50e8d8daR74).

Reply:
okay.